be/src/util/simd/bits.h

```diff
@@ -89,16 +92,31 @@ inline size_t count_zero_num(const int8_t* __restrict data, size_t size) {
     return num;
 }
 
-inline size_t count_zero_num(const int8_t* __restrict data, const uint8_t* __restrict null_map,
```

Comment:
why remove `__restrict` ?

Reply:
This function seems never used. Could we remove it?

Reply:
Because `__restrict` is redundant for `const` pointers. I see this function is called in `VRuntimeFilterWrapper::execute` and cannot be remove.